code/game/objects/items/defib.dm

```diff
@@ -550,7 +550,12 @@
 		var/total_brute	= 0
 		var/tplus = world.time - H.timeofdeath	//length of time spent dead
 		var/obj/item/organ/heart = H.getorgan(/obj/item/organ/heart)
-		if(do_after(user, 20, target = H)) //placed on chest and short delay to shock for dramatic effect, revive time is 5sec total
+		if(do_after(user, 15, target = H))
+			if(user.job == "Medical Doctor" || user.job == "Paramedic" || user.job == "Chief Medical Officer")
+				user.say("Clear!", forced = "defib")
+			else
+				return
```

Comment:
```suggestion
```
The return here will end the entire proc early. Not what you want.

With this change, if the if condition is not fulfilled, the next code will run, which is what we want.